src/VirtualPage.php

```diff
@@ -32,6 +32,12 @@ protected function init(): void
         parent::init();
 
         $this->cb = $this->add([Callback::class, 'urlTrigger' => $this->urlTrigger ?: $this->name]);
+        unset($this->{'urlTrigger'});
```

Comment:
this is a very bad hack to dedup `urlTrigger` property stored at 2 places, but let's keep is for now, related with https://github.com/atk4/core/pull/245 , otherwise this property can not be set from seed